File: app/FarertPre/FarertPre/FarertPre/FarertTableViewCell.m
//
//  FarertTableViewCell.m
//  alps_ios
//
//  Created by TAKEDA, Noriyuki on 2014/07/21.
//  Copyright (c) 2014年 TAKEDA, Noriyuki. All rights reserved.
//

#import "FarertTableViewCell.h"

@implementation FarertTableViewCell

@synthesize salesKm, lineName, stationName, fare;

- (id)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
{
    self = [super initWithStyle:style reuseIdentifier:reuseIdentifier];
    if (self) {
        // Initialization code
    }
    return self;
}

- (void)awakeFromNib
{
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated
{
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: app/Farert/Farert/FarertTableViewCell.swift
//
//  FarertTableViewCell.swift
//  Farert
//
//  Created by TAKEDA, Noriyuki on 2015/04/18.
//  Copyright (c) 2015年 TAKEDA, Noriyuki. All rights reserved.
//

import UIKit

class FarertTableViewCell: UITableViewCell {

    @IBOutlet weak var lineName: UILabel!
    @IBOutlet weak var stationName: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }
    override func setSelected(selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }

}
